src/Uno.UWP/Generated/3.0.0.0/Windows.System/MemoryManager.cs

```diff
@@ -1,21 +1,34 @@
 #pragma warning disable 108 // new keyword hiding
 #pragma warning disable 114 // new keyword hiding
+using System;
+using Android.App;
+using Android.Content;
+using Uno.UI;
+
 namespace Windows.System
 {
-	#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+	#if __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
 	[global::Uno.NotImplemented]
 	#endif
 	public  partial class MemoryManager 
 	{
-		#if __ANDROID__ || __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
-		[global::Uno.NotImplemented("__ANDROID__", "__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
+		#if __IOS__ || NET461 || __WASM__ || __SKIA__ || __NETSTD_REFERENCE__ || __MACOS__
+		[global::Uno.NotImplemented("__IOS__", "NET461", "__WASM__", "__SKIA__", "__NETSTD_REFERENCE__", "__MACOS__")]
 		public static ulong AppMemoryUsage
 		{
 			get
 			{
 				throw new global::System.NotImplementedException("The member ulong MemoryManager.AppMemoryUsage is not implemented in Uno.");
 			}
 		}
+		#else
+		public static ulong AppMemoryUsage
+		{
+			get
+			{
+				return (ulong) GC.GetTotalMemory(false);
```

Comment:
Are you sure that GC.GetTotalMemory(false) is equivalent of MemoryManager.AppMemoryUsage?
It seems to me that GC.GetTotalMemory returns number of bytes currently allocated in managed memory, i.e. bytes for data; and MemoryManager.AppMemoryUsage is data and code.